src/PlottingInterface.cc

```diff
@@ -80,19 +80,19 @@ class TransportPrivate
 class PlottingIfacePrivate
 {
   /// \brief Responsible for transport messages and topics
-  public: Transport *transport;
+  public: Transport transport;
 
   /// \brief current plotting time
-  public: double time;
+  public: double *time {new double};
```

Comment:
Is this variable needed? Can we just have `plottingTimeRef`?

Reply:
yeah great idea :D 
i modified it